apps/yarn.lock

```diff
@@ -4151,21 +4143,13 @@ firebase@^2.4.2:
   dependencies:
     faye-websocket ">=0.6.0"
 
-firmata@^0.11.3:
-  version "0.11.3"
-  resolved "https://registry.yarnpkg.com/firmata/-/firmata-0.11.3.tgz#e122e4598aa1c67cd82165bf0fb727e5b89314f4"
-  dependencies:
-    browser-serialport "*"
-    es6-shim "^0.33.6"
-    serialport "^2.0.1"
-
-firmata@^0.12.0:
-  version "0.12.0"
-  resolved "https://registry.yarnpkg.com/firmata/-/firmata-0.12.0.tgz#5d97dd122f5f77fec922604e074209d856d776b1"
+firmata@0.16.0, "firmata@>=0.11.3 <1.0", firmata@latest:
+  version "0.16.0"
+  resolved "https://registry.yarnpkg.com/firmata/-/firmata-0.16.0.tgz#6f2a759b1a98de234ced010fa3d4d9534d62b2e7"
   dependencies:
-    browser-serialport "*"
-    es6-shim "^0.33.6"
-    serialport "^3.0.0"
+    browser-serialport latest
+    es6-shim latest
+    serialport "^4.0.0"
```

Comment:
Ugh... we _already_ had two copies of `firmata` - we might have been getting lucky that the right version was being loaded when trying to interact with static state in that library.  I'm feeling a lot better about this now.  I'll open a PR with Rick upstream and suggest that `johnny-five` and `playground-io` ought to have the same (or closer) firmata dependencies, and we can update this again if he settles on a different version range.